src/View/Helper/ListsHelper.php

```diff
@@ -604,11 +604,13 @@ public function displayListsModule($allListsArray, $sentences)
                         $list['id']
                     )
                 );
-                 if(($list['user_id'] == $currentUserId && $list['editable_by'] != 'no_one') || $list['editable_by'] == 'anyone')
-                 {
-                    echo $this->_displayRemoveLink($list['id'], $sentenceId, $list['name']);
-                 }
-                echo '</li>';
+                if(currentUser::isMember()){
+                    if(($list['user_id'] == $currentUserId && $list['editable_by'] != 'no_one') || $list['editable_by'] == 'anyone')
+                    {
+                       echo $this->_displayRemoveLink($list['id'], $sentenceId, $list['name']);
+                    }
+                   echo '</li>';
```

Comment:
Having this line inside the if-block will result in an unclosed `li` element for guests. I guess most browsers will still be able to render the list correctly because they are used to fixing broken HTML code but I don't think we should rely on that.